plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcMetadata.java

```diff
@@ -370,8 +371,8 @@ public ConnectorTableMetadata getTableMetadata(ConnectorSession session, Connect
                 jdbcClient.getTableHandle(session, tableName)
                         .ifPresent(tableHandle -> columns.put(tableName, getTableMetadata(session, tableHandle).getColumns()));
             }
-            catch (TableNotFoundException e) {
-                // table disappeared during listing operation
+            catch (TableNotFoundException | AccessDeniedException e) {
+                // table disappeared during listing operation or user is not access it
```

Comment:
missing "allowed to"

Reply:
bump;

also add a comment why this is correct thing to do. 
current comment describes _the what_, but _the why_ is not obvious